pkg/cloud/cloud.go

```diff
@@ -796,6 +811,7 @@ func (c *cloud) CreateSnapshot(ctx context.Context, volumeID string, snapshotOpt
 		Description:       aws.String(descriptions),
 	}
 
+	lastCreateSnapshot.Store(volumeID, time.Now())
```

Comment:
May also need to delete entry at some point, otherwise memory consumed by driver would balloon.